src/Cargo.lock

```diff
@@ -1,7 +1,3 @@
-[root]
-name = "workspace_symbol"
-version = "0.1.0"
-
```

Comment:
No idea why this moved. I just did `cargo update -p jobserver` in the `src` dir.